functions/Add-DbaAgDatabase.ps1

```diff
@@ -71,7 +94,7 @@ function Add-DbaAgDatabase {
 
     .NOTES
         Tags: AvailabilityGroup, HA, AG
-        Author: Chrissy LeMaire (@cl), netnerds.net
+        Author: Chrissy LeMaire (@cl), netnerds.net / Andreas Jordan (@JordanOrdix), ordix.de
```

Comment:
```suggestion
        Author: Chrissy LeMaire (@cl), netnerds.net | Andreas Jordan (@JordanOrdix), ordix.de
```

Has to be pipe symbol for our doc generation to parse it correctly

Reply:
Thanks.